### ISSUE TITLE
roundup of post-code-push glitch reports on Discord

### ISSUE THREAD

**Issue by kareila** [dreamwidth/dw-free#2784]
1. Some people are saying the Tropo Purple colors are too bright/saturated in Foundation. Not sure if this is something we need to revise.

2. Show more/less button on comment preview displays even if the comment is short enough to be already fully shown.

3. Foundation default table styling applies to tables in journal entries on site schemed pages.

4. Casual HTML dropdown is bright white in Gradation Vertical.

**Comment by kareila** [comment]
Here's an example of (3) from support: https://www.dreamwidth.org/support/see_request?id=41811

**Comment by rahaeli** [comment]
* the table holding the pagination links on entries that have 50+ comments used to have a background of the skin's background accent color, now it's the default page bg

**Comment by rahaeli** [comment]
* Tropo Red is having at least partially the same problem as in Tropo Purple on comment headers, and I think what's wrong *may be* that the primary accent background color and secondary accent background color have been swapped? The background in comment headers on screened comments in Red is the light-pink secondary accent background color now, not the darker red primary accent bg color, at least.

**Comment by rahaeli** [comment]
* Wide images in inbox notifications are forcing sidescroll/stretch (and the user implies that they weren't before)

**Comment by rahaeli** [comment]
* There's a collection of people who are complaining that their CSS in entries/comments is having new and different padding applying to it (and breaking their CSS designs) and that the padding in comment headers is inadequate now (and I agree it looks a bit wonky), so I'm guessing there's an order-of-load issue going on somewhere maybe? This is probably bigger than just a single bullet point in a catch-all issue, and will need to be broken out into its own issue with some test cases, but I'm trying to make sure I dump all the problems people are reporting somewhere

**Comment by kareila** [comment]
Textarea boxes are width: 100% and should be width: auto - https://dw-maintenance.dreamwidth.org/83294.html?thread=3363678#cmt3363678

**Comment by kareila** [comment]
I think we determined that emailed notifications of entries and/or comments didn't have linebreaks applied correctly? I haven't seen an example of this yet.

**Comment by kareila** [comment]
ilyena_sylph provided this screenshot, which I believe shows an overly narrow reply div on the reply page (as opposed to quick reply). Clicking "reply at the webpage" from an email links to this view. Journal style is Abstractia Abyss.

![unknown](https://user-images.githubusercontent.com/1976742/88492863-83f0b380-cf73-11ea-8be9-2589e7400d27.png)

**Comment by nfagerlund** [comment]
- The person browsing with some kind of antique potato-browser with the three-line-high comment form... we can maybe fix that by setting height: auto on the comment textarea. Worth a try. (foundation sets height of 2.4 rems or so for textareas (?????), then sets height: auto if it's a textarea with a `rows` attr. I think their browser doesn't understand attr selectors.)

**Comment by rahaeli** [comment]
I was gonna pass that along in a few days when things had calmed down somewhat; worth a try if you think it might be that easy!

**Comment by nfagerlund** [comment]
> - Wide images in inbox notifications are forcing sidescroll/stretch (and the user implies that they weren't before)

Uhhhhh I'm pressing the "doubt" button on that one. Nothing would have prevented this previously, and nothing we did recently would have changed that. 

More likely, they just never saw it before because anyone posting an extra-wide image would also blow out the comment page itself, and now people don't have to worry about that.

**Comment by rahaeli** [comment]
Yeah, I only reported it because it implied that we missed the inbox in the "stop wide images from blowing things out" change, not because I thought things had changed, heh.

**Comment by nfagerlund** [comment]
The “reply from suspended user” message should be bold, and it’s not. (There’s one in the call for PSV volunteers maint post.)

**Comment by nfagerlund** [comment]
The new skins fade out the font color of h2 through h6 headers, slightly. (Darken 10% on gradation, lighten 5% on celerity, lighten 20% on tropos.) I don't know why! But for comment headers, since they're against a colored background, that makes them look "faded" compared to previously.

**Comment by kareila** [comment]
> The new skins fade out the font color of h2 through h6 headers, slightly. (Darken 10% on gradation, lighten 5% on celerity, lighten 20% on tropos.) I don't know why! But for comment headers on Tropo, since they're against a colored background, that makes them look "faded" compared to previously.
> 
> The other skins don't look as bad. So maybe just back off the header lightening on the Tropos.

This was fixed in dreamwidth/dw-nonfree#139

> * the table holding the pagination links on entries that have 50+ comments used to have a background of the skin's background accent color, now it's the default page bg

This was fixed in dreamwidth/dw-nonfree#140 (at least for Tropo; #2801 should get dw-free skins)

**Comment by nfagerlund** [comment]
What's the _desired_ layout for abstractia? It looks like there's a couple things going on here: 

- It's expecting everything to be content-box sizing, but new reply forms set all their contents to border-box to make sure things behave predictably. 
- It's setting its own widths and paddings and margins for both the quickreply and the talkform. 
- The talkform now uses the quickreply's layout, but it still has a couple of unique IDs on its HTML elements. Abstractia targets some of those IDs (at least `postform`) to set styles on the _old_ talkform, which means it's using wrong assumptions to style the new one. 

It looks like abstractia wants the reply form box to match the width of the content container above it, since they both go into shaded boxes, but I don't know that style From Olden Days so I'm not 100% sure.
